src/components/Button/Button.js

```diff
@@ -23,48 +23,47 @@ const propTypes = {
   onClick: PropTypes.func.isRequired,
 };
 
+const NOOP = () => {};
+
 const Button = props => {
   const [isElementDisabled, customOverrides] = useSelector(state => [
     selectors.isElementDisabled(state, props.dataElement),
     selectors.getCustomElementOverrides(state, props.dataElement),
   ]);
 
-  props = { ...props, ...customOverrides };
   const {
+    disable,
     isActive,
     mediaQueryClassName,
     img,
     label,
     color,
     dataElement,
-    onClick = () => {},
+    onClick = NOOP,
     className,
     title,
-  } = props;
+  } = { ...props, ...customOverrides };
 
-  const buttonClass = classNames({
-    Button: true,
-    active: isActive,
-    [mediaQueryClassName]: mediaQueryClassName,
-    [className]: className,
-  });
-  const isBase64 = img && img.trim().indexOf('data:') === 0;
+  const isBase64 = img?.trim().startsWith('data:');
   // if there is no file extension then assume that this is a glyph
   const isGlyph =
-    img && !isBase64 && (img.indexOf('.') === -1 || img.indexOf('<svg') === 0);
-
-  let content;
-  if (isGlyph) {
-    content = <Icon glyph={img} color={color} />;
-  } else if (img) {
-    content = <img src={img} />;
-  } else if (label) {
-    content = <p>{label}</p>;
-  }
+    img && !isBase64 && (!img.includes('.') || img.startsWith('<svg'));
 
   const children = (
-    <div className={buttonClass} data-element={dataElement} onClick={onClick}>
-      {content}
+    <div
+      className={classNames({
+        Button: true,
+        active: isActive,
+        disable,
+        [mediaQueryClassName]: mediaQueryClassName,
+        [className]: className,
+      })}
+      data-element={dataElement}
+      onClick={disable ? NOOP : onClick}
```

Comment:
assuming we go with the way of making it have an initial value of undefined.
can we do:
`!disable && onClick` ?

Reply:
I answered this in another comment